### PR TITLE
chore: preparing to open the sources

### PR DESCRIPTION
**Pull request by zavgorodnii**
This PR:

1. Adds errors when non-contract addresses try to execute most of the handlers, and fixed a lot of related tests;
2. Updated packet timeout (set to 30 days);
3. Removed tx CLI entries & some of the icq CLI entries;
4. A lot of formatting (imports + removed all scaffolding comments);
5. Fixed `CmdInterchainAccountCmd()` implementation.

**HOW TO TEST THIS CODE?**

Just use the usual 3-terminals setup & run all `test_*.sh` scripts from the contracts repository.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -205,10 +193,9 @@ var (
 		transfer.AppModuleBasic{},
 		vesting.AppModuleBasic{},
 		wasm.AppModuleBasic{},
-		//monitoringp.AppModuleBasic{},
+		// monitoringp.AppModuleBasic{},
```
Comment (ratik) [inline app/app.go:196]: should we just remove unused imports?

File: app/genesis.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 )
 
-// The genesis state of the blockchain is represented here as a map of raw json
+// GenesisState is the genesis state of the blockchain is represented here as a map of raw json
```
Comment (ratik) [inline app/genesis.go]:
```suggestion
// GenesisState is the genesis state of the blockchain represented here as a map of raw json
```

File: internal/sudo/sudo.go [hunk 3]
```diff
@@ -224,22 +234,21 @@ func (s *SudoHandler) SudoTxQueryResult(
...
-	x := SudoMessageTxQueryResult{}
+	x := MessageTxQueryResult{}
 	x.TxQueryResult.QueryID = queryID
 	x.TxQueryResult.Height = uint64(height)
 	x.TxQueryResult.Data = data
 
 	m, err := json.Marshal(x)
 	if err != nil {
-		s.Logger(ctx).Error("failed to marshal SudoMessageTxQueryResult message",
+		s.Logger(ctx).Error("failed to marshal MessageTxQueryResult message",
```
Comment (ratik) [inline internal/sudo/sudo.go]:
```suggestion
		s.Logger(ctx).Error("SudoTxQueryResult: failed to marshal MessageTxQueryResult message",
```

File: internal/sudo/sudo.go [hunk 4]
```diff
@@ -254,27 +263,26 @@ func (s *SudoHandler) SudoTxQueryResult(
...
+func (s *Handler) SudoKVQueryResult(
 	ctx sdk.Context,
 	contractAddress sdk.AccAddress,
 	queryID uint64,
 ) ([]byte, error) {
 	s.Logger(ctx).Info("SudoKVQueryResult", "contractAddress", contractAddress)
 
-	// TODO: basically just for unit tests right now. But i think we will have the same logic in the production
 	if !s.wasmKeeper.HasContractInfo(ctx, contractAddress) {
 		s.Logger(ctx).Debug("contract was not found", "contractAddress", contractAddress)
```
Comment (ratik) [inline internal/sudo/sudo.go]:
```suggestion
		s.Logger(ctx).Debug("SudoKVQueryResult: contract was not found", "contractAddress", contractAddress)
```

File: app/app.go (hunk 1 above)
Comment (zavgorodnii) [reply app/app.go:196]: https://p2pvalidator.atlassian.net/browse/LSC-138

Comment (mmikev) [inline app/app.go:196]: commented line

File: app/app.go [hunk 5]
```diff
@@ -280,15 +267,14 @@ type App struct {
...
 
 	// make scoped keepers public for test purposes
-	ScopedIBCKeeper        capabilitykeeper.ScopedKeeper
-	ScopedTransferKeeper   capabilitykeeper.ScopedKeeper
-	ScopedMonitoringKeeper capabilitykeeper.ScopedKeeper
-	ScopedWasmKeeper       capabilitykeeper.ScopedKeeper
-	ScopedInterTxKeeper    capabilitykeeper.ScopedKeeper
+	ScopedIBCKeeper      capabilitykeeper.ScopedKeeper
+	ScopedTransferKeeper capabilitykeeper.ScopedKeeper
+	// ScopedMonitoringKeeper capabilitykeeper.ScopedKeeper
```
Comment (mmikev) [inline app/app.go:272]: commented line

File: app/app.go [hunk 6]
```diff
@@ -570,11 +552,10 @@ func New(
 		ibc.NewAppModule(app.IBCKeeper),
 		params.NewAppModule(app.ParamsKeeper),
 		transferModule,
-		//monitoringModule,
+		// monitoringModule,
```
Comment (mmikev) [inline app/app.go:555]: commented line

File: app/app.go (hunk 5 above)
Comment (zavgorodnii) [reply app/app.go:272]: https://p2pvalidator.atlassian.net/browse/LSC-138

File: app/app.go (hunk 6 above)
Comment (zavgorodnii) [reply app/app.go:555]: https://p2pvalidator.atlassian.net/browse/LSC-138

File: x/interchainqueries/keeper/msg_server.go [hunk 7]
```diff
@@ -169,30 +169,30 @@ func (k msgServer) SubmitQueryResult(goCtx context.Context, msg *types.MsgSubmit
...
 			}
+		}
 
-			queryOwner, err := sdk.AccAddressFromBech32(query.Owner)
-			if err != nil {
-				ctx.Logger().Error("SubmitQueryResult: failed to decode AccAddressFromBech32",
-					"error", err, "query", query, "message", msg)
-				return nil, sdkerrors.Wrapf(types.ErrInternal, "failed to decode owner contract address: %v", err)
-			}
+		queryOwner, err := sdk.AccAddressFromBech32(query.Owner)
```
Comment (mmikev) [inline x/interchainqueries/keeper/msg_server.go]: You decode queryOwner twice: here and on [L104](https://github.com/neutron-org/neutron/blob/2cc6b752dd5ffdbc1d2f118d1edd6a52fc46dd0c/x/interchainqueries/keeper/msg_server.go#L104)

Comment (zavgorodnii) [reply x/interchainqueries/keeper/msg_server.go]: fixed

File: testutil/test_helpers.go [hunk 8]
```diff
@@ -70,6 +72,26 @@ func (suite *IBCConnectionTestSuite) GetNeutronZoneApp(chain *ibctesting.TestCha
 	return testApp
 }
 
+func (suite *IBCConnectionTestSuite) StoreReflectCode(ctx sdk.Context, addr sdk.AccAddress, path string) uint64 {
+	// wasm file built with https://github.com/neutron-org/neutron-contracts/tree/main/contracts/reflect
```
Comment (swelf19) [inline testutil/test_helpers.go:76]: comment is a bit confusing, "path" variable is not a contract witch built with particular repo. 
Do we need a path argument at all, taking into account, method name is Store**Reflect**Code

**Review by pr0n00gler — APPROVED** [review]

File: x/interchainqueries/keeper/msg_server.go [hunk 9]
```diff
@@ -33,6 +33,17 @@ func (k msgServer) RegisterInterchainQuery(goCtx context.Context, msg *types.Msg
...
 
+	senderAddr, err := sdk.AccAddressFromBech32(msg.Sender)
+	if err != nil {
+		k.Logger(ctx).Debug("RegisterInterchainQuery: failed to parse sender address", "sender_address", msg.Sender)
+		return nil, fmt.Errorf("failed to parse sender address: %v", err)
+	}
+
+	if !k.wasmKeeper.HasContractInfo(ctx, senderAddr) {
+		k.Logger(ctx).Debug("RegisterInterchainQuery: contract not found", "sender_address", msg.Sender)
+		return nil, fmt.Errorf("%s is not a contract address", msg.Sender)
```
Comment (swelf19) [inline x/interchainqueries/keeper/msg_server.go]: why not sdkerrors? at least for uniformity

**Review by swelf19 — APPROVED** [review]

**Pull request merged**